pubsub/responses.go

```diff
@@ -16,16 +16,61 @@ package pubsub
 // AppResponseStatus represents a status of a PubSub response.
 type AppResponseStatus string
 
+// BulkPublishStatus represents a status of a Bulk Publish response.
+type BulkPublishStatus string
+
 const (
 	// Success means the message is received and processed correctly.
 	Success AppResponseStatus = "SUCCESS"
 	// Retry means the message is received but could not be processed and must be retried.
 	Retry AppResponseStatus = "RETRY"
 	// Drop means the message is received but should not be processed.
 	Drop AppResponseStatus = "DROP"
+	// PublishSucceeded represents that message was published successfully.
```

Comment:
```suggestion
	// PublishSucceeded means that message was published successfully.
```
Only for consistency.